transcoder/message/handler/SequencerHandler.py

```diff
@@ -41,6 +41,6 @@ def append_manufactured_fields(self, schema: DatacastSchema):
         schema.fields.append(MessageHandlerIntField(self.sequence_number_field_name))
 
     def handle(self, message: ParsedMessage):
-        if message.dictionary is not None:
+        if message.ignored is False:
```

Comment:
I would check this, the message.dictionary is not None check may be needed still? Unless the upstream logic is never returning None, but I think it does on an exception.